schema/ext/vulnerability-1.0.xsd

```diff
@@ -10,6 +10,7 @@
 
     <xs:simpleType name="severityType" final="restriction">
         <xs:restriction base="xs:normalizedString">
+            <xs:enumeration value="None"/>
```

Comment:
I think 'Unknown' may be a better term. 'None' could be confused to mean a INFO level warning rather than a vulnerability of unknown severity.

Reply:
> I think 'Unknown' may be a better term. 'None' could be confused to mean a INFO level warning rather than a vulnerability of unknown severity.


'None' came from the cvss ratings.  My understanding is 'None' means there’s next to no threat.  'Unknown' would be the score hasn’t been calculated.  

![Screen Shot 2019-09-06 at 2 22 32 PM](https://user-images.githubusercontent.com/4838322/64531990-9cfe6080-d2de-11e9-9982-0d174a8e83e6.png)

Reply:
>'None' came from the cvss ratings. My understanding is 'None' means there’s next to no threat. 'Unknown' would be the score hasn’t been calculated.

I've added `Unknown` the list of options as well. c3af551